src/main/webapp/app/markdown-editor/specialCommands/specialCommand.ts

```diff
@@ -0,0 +1,6 @@
+import { Command } from 'app/markdown-editor/commands/command';
+
+export abstract class SpecialCommand extends Command {
```

Comment:
We said we want to rename SpecialCommand to DefaultCommand.
Unfortunately there is still a documentation missing here :-(